src/core/Expander/Expander.tsx

```diff
@@ -61,15 +65,55 @@ export class Expander extends Component<ExpanderProps> {
       this.props.defaultOpen !== undefined ? this.props.defaultOpen : false,
   };
 
+  componentDidUpdate(prevProps: ExpanderProps, prevState: ExpanderState) {
```

Comment:
We should really consider ditching the component level for this kind of components or at least somehow getting rid of the duplicate logic. I guess there is not much we can do with this for the time being / within this ticket, but this is something to keep in mind.

Reply:
And please consider all comments in component level for core level as well.

Reply:
Could not find any comments that I have added in this PR. Which one did you have in mind?

Reply:
Not sure anymore. I guess it's fine.